File: HFY.Entities/Concrete/LastikMarkaDesenOzellikler.cs
using HFY.Core.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace HFY.Entities.Concrete
{
    public class LastikMarkaDesenOzellikler : IEntity
    {
        [Key]
        public int LastikMarkaDesenOzellikID { get; set; }

        public int LastikMarkaDesenID { get; set; }
        public decimal DisDerinligi { get; set; }
        public byte KatOrani { get; set; }
        public int EbatID { get; set; }
        public int Sira { get; set; }
        public int OlusturanId { get; set; }

        public DateTime OlusturmaTarihi { get; set; }

        public int DuzenleyenId { get; set; }

        public DateTime DuzenlemeTarihi { get; set; }

        public bool Aktif { get; set; }

        public bool ListeAktiflik { get; set; }
    }
}

File: HFY.MvcWebUI.Angular/ClientApp/src/app/lazy-general-page/classes/lastik-marka-desen-ozellikler/lastik-marka-desen-ozellikler.ts
export class LastikMarkaDesenOzellikler {
    LastikMarkaDesenOzellikID: number;

    LastikMarkaDesenID: number;
    DisDerinligi: number;
    KatOrani: number;
    EbatID:number;
    Sira: number;
    OlusturanId: number;
    OlusturmaTarihi: Date | string;
    DuzenleyenId: number;
    DuzenlemeTarihi: Date | string;
    Aktif: boolean;
    ListeAktiflik: boolean;

    jqueryDataTableLastikMarkaDesenOzellikler() {
        return [ { data: 'DisDerinligi' }, { data: 'KatOrani' }, { data: 'Sira' }, { data: 'EbatID' },
        { data: 'Aktif' }]
    }
}
